src/Nethermind/Nethermind.Facade/Eth/EthSyncingInfo.cs

```diff
@@ -55,18 +63,33 @@ public SyncingResult GetFullInfo()
             if (_syncConfig.FastSync)
             {
                 if (_syncConfig.DownloadReceiptsInFastSync &&
-                    _receiptStorage.LowestInsertedReceiptBlockNumber > _syncConfig.AncientReceiptsBarrierCalc)
+                    (_receiptStorage.LowestInsertedReceiptBlockNumber == null || _receiptStorage.LowestInsertedReceiptBlockNumber > _syncConfig.AncientReceiptsBarrierCalc))
                 {
-                    return new SyncingResult { IsSyncing = true };
+                    if (_logger.IsInfo) _logger.Info($"Receipts not finished - EthSyncingInfo - HighestBlock: {bestSuggestedNumber}, CurrentBlock: {headNumberOrZero}, AncientReceiptsBarrier: {_syncConfig.AncientReceiptsBarrierCalc}. LowestInsertedBodyNumber: {_blockTree.LowestInsertedBodyNumber} LowestInsertedReceiptBlockNumber: {_receiptStorage.LowestInsertedReceiptBlockNumber}");
+                    return new SyncingResult
+                    {
+                        CurrentBlock = headNumberOrZero,
+                        HighestBlock = bestSuggestedNumber,
+                        StartingBlock = 0L,
```

Comment:
Shouldn't we set `StartingBlock` - in all cases to `headNumberOrZero` if we are in FastSync?

Reply:
hmmm I don't think we have good reasons for doing breaking change in `StartingBlock`. I have no idea what is semantics of this field.